notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb

```diff
@@ -404,7 +363,7 @@
       "source": [
         "### Initialize Vertex AI and BigQuery SDKs for Python\n",
         "\n",
-        "Initialize the Vertex AI SDK for Python for your project and corresponding bucket."
+        "Initialize the Vertex AI SDK and BigQuery SDK for Python with your project and the created bucket."
```

Comment:
change to: (to align with product naming:
- "Initialize the Vertex AI SDK for Python and BigQuery SDK with your project and the created bucket."